habitat/plan.sh

```diff
@@ -15,9 +15,17 @@ pkg_deps=(
   core/libxslt
   core/net-tools
   core/ruby
+
+  # Needed for some InSpec resources
+  core/bind
+  core/curl
+  core/docker
+  core/less
+  core/mysql-client
+  core/netcat
+  core/postgresql
```

Comment:
Do we need a full pg server or just the clients?

Reply:
Habitat doesn't currently have a package that contains only the clients. Could be worth making one though...